lib/usdUfe/ufe/UsdClipboardCommands.cpp

```diff
@@ -480,20 +497,62 @@ void UsdPasteClipboardCommand::execute()
         }
 
         if (!clipboardPrims.empty()) {
-            auto duplicateCmd
-                = UsdUndoDuplicateSelectionCommand::create(clipboardPrims, dstParentItem);
-            duplicateCmd->execute();
-            appendToVector(duplicateCmd->targetItems(), pasteInfo.successfulPastes);
-            auto tmpTargetItems = duplicateCmd->targetItems();
-            _targetItems.insert(_targetItems.end(), tmpTargetItems.begin(), tmpTargetItems.end());
-            _itemsToSelect.insert(
-                _itemsToSelect.end(), tmpTargetItems.begin(), tmpTargetItems.end());
+            UsdSceneItem::Ptr pasteTarget = dstParentItem;
+            if (pasteAsSibling) {
+                auto hier = Ufe::Hierarchy::hierarchy(dstParentItem);
+                auto parentItem = hier ? hier->parent() : nullptr;
+                auto newPasteTarget = downcast(parentItem);
+                if (newPasteTarget) {
+                    pasteTarget = newPasteTarget;
+                } else {
+                    pasteAsSibling = false;
+                }
+            }
```

Comment:
When pasting as sibling get the parent of the destination item so when pasting the new prim will be a sibling.

Reply:
Only applies to regular prims and not materials or shaders.